controls/cis_ocp_1_3_0/section-1.yml

```diff
@@ -319,8 +319,13 @@ controls:
     controls:
     - id: 1.3.1
       title: Ensure that garbage collection is configured as appropriate
-      status: pending
-      rules: []
+      status: partial
+      rules: 
+        - terminated-pod-gc-threshold
```

Comment:
Note that in general the rule names in CaC use underscores, exactly as the directory name in the `applications/openshift` folder. What you used is probably the name from the CO which is different because the CO rules follow the kubernetes naming scheme which must be DNS-like and therefore doesn't allow underscores.

But because you are removing the rule, the `rules` attribute should be empty here.